### PR TITLE
CVSB-6510 Font-increase accessibility

### PR DESCRIPTION
**Pull request by agoncearenco**
This branch has been created from CVSB-6426 and will be merged into develop after CVSB-6426. ~~Therefore, the only commits that should be considered are 11a7953 and ed60828.~~

The 2 main reasons for this are the following:
1) CVSB-6426 introduces many new changes which I also made accessible. If I had made these changes on develop directly, it would need additional work after the CVSB-6426 merge.
2) If I made these changes on develop directly, it would introduce many conflicts which I could avoid by applying the changes to CVSB-6426

### CODE REVIEW

**Comment by vladddragomir** [comment]
i will continue reviewing this just after 6426 will be merged into develop because it's a huge PR otherwise.

File: src/pages/test-station/test-station-details/test-station-details.ts [hunk 1]
```diff
@@ -106,13 +108,15 @@ export class TestStationDetailsPage implements OnInit {
...
-      subTitle: `Confirm that you are at ${this.testStation.testStationName} (${this.testStation.testStationPNumber}) and that it is suitable to begin testing before continuing.`,
+      message: `Confirm that you are at ${this.testStation.testStationName} (${this.testStation.testStationPNumber}) and that it is suitable to begin testing before continuing.`,
+      cssClass: this.appService.isAccessibilityTextZoomEnabled()?'accessibility-limit-message-height':'',
       buttons: [
         {
           text: APP_STRINGS.CONFIRM,
           cssClass: 'bold-action-button',
           handler: () => {
             this.confirmStartVisit();
+            console.log(this.appService.isAccessibilityTextZoomEnabled())
```
Comment (KHodderDVSA) [inline src/pages/test-station/test-station-details/test-station-details.ts]: still needed?

Comment (agoncearenco) [reply src/pages/test-station/test-station-details/test-station-details.ts]: No, will remove

File: src/pages/testing/test-creation/test-review/test-review.ts [hunk 2]
```diff
@@ -88,7 +89,8 @@ export class TestReviewPage implements OnInit {
               private authService: AuthService,
               private store$: Store<LogsModel>,
               private firebaseLogsService: FirebaseLogsService,
-              private activityService: ActivityService) {
+              private activityService: ActivityService,
+              public appService: AppService) {
```
Comment (KHodderDVSA) [inline src/pages/testing/test-creation/test-review/test-review.ts]: Is this used anywhere?

File: src/pages/test-station/test-station-details/test-station-details.ts (hunk 1 above)
Comment (agoncearenco) [reply src/pages/test-station/test-station-details/test-station-details.ts]: Requested change completed.

**Pull request merged**